apps/music/test/unit/metadata/mock_album_art_cache.js

```diff
@@ -0,0 +1,19 @@
+
+/* exported MockAlbumArtCache */
+'use strict';
+
+
+var MockAlbumArtCache = {
+  getThumbnailURL: function() {
+    // Mock: return a fake promise.
+    return {
+      then: function() { }
```

Comment:
Can you use a real Promise here? That is, `return Promise.resolve();`. That'll probably be less error-prone in the future (I just got bitten by this with the `MockLazyLoader`, which I had to fork for the music app).